test-project-unconfigured/package-lock.json

```diff
@@ -1,8 +1,5917 @@
 {
   "name": "test-project-unconfigured",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Since the integration tests use `node:lts`, I upgraded these lockfiles to speed up `npm ci`.